BaragonAgentService/src/main/java/com/hubspot/baragon/agent/handlebars/RackMethodsHelper.java

```diff
@@ -0,0 +1,65 @@
+package com.hubspot.baragon.agent.handlebars;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+
+import com.hubspot.baragon.models.UpstreamInfo;
+
+public class RackMethodsHelper {
+
+  public static class RackMethods {
```

Comment:
A cleaner way to do this might be to un-nest the static class and just do `public static class RackMethodsHelper {`, and then have `generateAllRacks()` accept `Collection<UpstreamInfo> upstreams` directly.=